joomla/joomla-cms
# Fix chevron color in module menu assignment

Pull Request for Issue # .

### Summary of Changes
The chevron in the Module menu Assignment in light mode is not visible. In dark mode the chevron is visible.

![module-light-mode](https://github.com/joomla/joomla-cms/assets/5610413/8ec7d26b-aeed-4d39-a8e8-145ee2356ad7)
![module-dark-mode](https://github.com/joomla/joomla-cms/assets/5610413/cf4c6361-2a0e-43af-afce-3fbf6be0636d)

I used the default link color as new color.

### Testing Instructions
- Checkout this PR and build the CSS with: npm run build:css
- Set the template to light mode
- Check if the chevron is visible

### Actual result BEFORE applying this Pull Request
Chevron not visible

### Expected result AFTER applying this Pull Request
Chevron is visible

![module-light-mode-fixed](https://github.com/joomla/joomla-cms/assets/5610413/80f5776d-9c87-48e9-bb7a-83642adb5690)
